src/domain/hanke/accessRights/AccessRightsView.tsx

```diff
@@ -192,6 +207,16 @@ function AccessRightsView({ hankeUsers, hankeTunnus, hankeName, signedInUser }:
     );
   }
 
+  function getUserRoles(args: HankeUser) {
+    return (
+      <div>
+        {args.roolit.map((role) => (
+          <div>{t(`hankeUsers:roleLabels:${role}`)}</div>
```

Comment:
According to Figma roles should be separated by comma: https://www.figma.com/file/CSWoUcI2jslxfOgBYgjKuK/Haitaton-HDS-3.0.0?type=design&node-id=1154-13628&mode=design&t=mbEgdg4dADSdU3hI-4.

Reply:
How did I miss that! But, it also puts "Toteuttaja" before "Rakennuttaja". Is there some logic in this or could they be in alphabetical order (e.g. "Rakennuttaja, Toteuttaja")?

Reply:
I'm not sure about the order. I don't remember that there would have been any discussion about that.

Reply:
According to UX the order should be Omistaja, Rakennuttaja, Toteuttaja, Muu. I will implement this.

Reply:
Ok, that's good.

Reply:
Looks good and works as expected.